### DISCUSSION TITLE
[NEW RELEASE] Introducing MARS5, open-source, insanely prosodic text-to-speech (TTS) model.

### DISCUSSION THREAD

**Discussion by akshhack** [dlutton/flutter_tts#505] [Show and tell]
![mars-banner](https://github.com/dlutton/flutter_tts/assets/21692676/135f9533-6d71-4fc5-bc73-ae0631143be9)

Hey community members! 👋 

We, at [CAMB.AI](https://www.camb.ai), are super stoked to announce the open source release of **MARS5**, a new speech emulation model that is able to replicate even extremely tough prosody like sports commentary, anime, movies with just a few seconds of audio reference. 

_Check out our release:_ https://github.com/Camb-ai/MARS5-TTS

Watch our demo here:

https://github.com/dlutton/flutter_tts/assets/21692676/096ea91e-bc6d-419a-9605-12b0eda9f499

and the full release video: https://www.youtube.com/watch?v=bmJSLPYrKtE

We're excited to hear feedback and see the community build on top of it!

Quick links:
**Discord:** https://discord.gg/ZzsKTAKM
**Github:** https://www.github.com/camb-ai/mars5-tts
**Website:** https://www.camb.ai/
**Youtube:** https://www.youtube.com/@camb-ai

**Comment by echogit** [comment]
Can it be embedded in flutter? And used as a standalone engine?
